test/sql-server-cdc/70-lsn-updated-during-snapshot.td

```diff
@@ -0,0 +1,86 @@
+# Copyright Materialize, Inc. and contributors. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+
+#
+# Test that if we snapshot while writes are happening we will not lose data.
+# We want to guarantee that there were writes to t2 while t0 and t1 are being
+# snapshotted so we write 3 new rows every 10ms to t2.
+# We then halt the job writing rows to t2. Count the number of rows in SQL Server
+# then assert that the final number of rows in t2 are the same.
+#
+# Thus we verify that t2 picks the correct LSN to rewind from after snapshotting.
+#
+# Note: we chose 10ms as that was found to be the lowest interval that SQL Server would
+# reliably execute a job.
```

Comment:
Is this test validating something more than `workflow_snapshot_consistency`?

Reply:
yes, `workflow_snapshot_consistency` validates for one table. here we are trying to force the lsn to be incremented while t3 and t4 are being snapshotted. I updated the comment and to make it clearer and to actually test what I set out to test yesterday. we are forcing the snapshot_lsn and rewinds to be selected per table rather than for all tables.

Reply:
but I'm also happy to reimplement into that workflow using python to be more consistent.